tests/behavior/golden_goal_striker_penalized.lua

```diff
@@ -32,41 +32,40 @@ function on_cycle()
         }
     end
 
-    if state.cycle_count == 200 then
+    if state.cycle_count == 100 then
```

Comment:
Does this test multiple features?
Its a golden goal with a penalized striker and remains as such. The only difference is that now the striker is penalized from the beginning instead of about three seconds after the kick off, making the scenario more realistic.